.goreleaser.yml

```diff
@@ -15,37 +13,18 @@ builds:
       - -X keboola-as-code/src/build.BuildVersion={{ if index .Env "TARGET_VERSION"  }}{{ .Env.TARGET_VERSION }}{{ else }}dev{{ end }}
       - -X keboola-as-code/src/build.BuildDate={{.Date}}
       - -X keboola-as-code/src/build.GitCommit={{.Commit}}
-    ignore:
-      - goos: windows
-        goarch: arm64
+    hooks:
+      # Copy binary to TARGET_PATH if set
+      post: '{{ if index .Env "TARGET_PATH"  }}cp "{{ .Path }}" "{{ .Env.TARGET_PATH }}"{{ else }}true{{ end }}'
 
 changelog:
-  # Set it to true if you wish to skip the changelog generation.
-  # This may result in an empty release notes on GitHub/GitLab/Gitea.
   skip: true
 
 archives:
-  - format_overrides:
-      - goos: "windows"
-        format: "zip"
-    id: kbc
-    # dont package README and LICENSE files to the release archive
-    files:
+  - files:
       - none*
-    builds:
-      - kbc
-    name_template: "kbc-v{{ .Version }}-{{ .Os }}_{{ .Arch }}"
-    replacements:
-      amd64: 64-bit
-      "386": 32-bit
-      darwin: macOS
-      linux: linux
+    format: zip
```

Comment:
Formaz `zip` je podporovany vsade, ... asi zbytocne tam budeme pridavat podmienku pre Linux a `tar.gz`, kedze si to kazdy aj tak nainstaluje cez package manager.